api/ElementInternals.json

```diff
@@ -2048,14 +2048,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "121",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.element.customstateset.enabled",
-                  "value_to_set": "true"
-                }
-              ]
+              "version_added": "126"
```

Comment:
Part of the CustomStateSet stuff - matches the bugzilla issue. Other docs work in https://github.com/mdn/content/issues/33018